src/lib/isAbaRouting.js

```diff
@@ -0,0 +1,20 @@
+import assertString from './util/assertString';
+
+// http://www.brainjar.com/js/validation/
+const isRoutingReg = /^[0-9]{9}$/;
+
+export default function isAbaRouting(str) {
+  assertString(str);
+  str = str.trim();
+
+  if (!isRoutingReg.exec(str)) return false;
```

Comment:
The `exec()` method returns an array with the matched capture groups and indexes but,
since we only want to know, if the regexp matches at all (true/false) and don't really care about the matched content, using the `test()` method should be better here.
What do you think?

`isRoutingReg.exec(str)`->  `isRoutingReg.test(str)`

Reply:
Hi Panagiotis,

All your proposals are very reasonable. I will implement the changes.

Thank you very much for your review!